qabot/pipeline_maintenance.py

```diff
@@ -545,22 +470,6 @@ def replay_nightly_run(self, labels=""):
 
 
 if __name__ == "__main__":
-    # pipem = PipelineMaintenance()
-    # result = pipem.failure_rate_for_test_suite("test-portal-homepageTest")
-    # result = pipem.quarantine_ci_env("jenkins-new")
-    # result = pipem.check_pool_of_ci_envs()
-    # result = pipem.ci_benchmarking("cdis-manifest", "3265", "K8sReset")
-    # result = pipem.ci_benchmarking("gitops-qa", "1523", "RunTests")
-    # negative test
-    # result = pipem.ci_benchmarking("gen3-qa", "666", "Typo")
-    # result = pipem.fetch_ci_failures("gitops-qa", 1649)
-    # result = pipem.fetch_ci_failures("gen3-qa", 700)
-    # result = pipem.create_bug_ticket(
-    #    title="this PR-123 is failing",
-    #    description="help, this is failing",
-    #    assignee="Atharva Rane",
-    # )
-    # result = pipem.get_repo_sme("arborist")
     pipem1 = PipelineMaintenance()
     pipem1.react_to_jenkins_updates(
```

Comment:
can we use a function that is present in code now, since we are removing jenkins functions